tests/core/pyspec/eth2spec/test/bellatrix/block_processing/test_process_execution_payload.py

```diff
@@ -379,7 +379,7 @@ def run_randomized_non_validated_execution_fields_test(spec, state, rng, executi
     next_slot(spec, state)
     execution_payload = build_randomized_execution_payload(spec, state, rng)
 
-    if is_post_eip7732(spec):
+    if is_post_gloas(spec):
```

Comment:
All tests that call `run_randomized_non_validated_execution_fields_test` run except Gloas. Thus, it seems `run_randomized_non_validated_execution_fields_test` doesn't need `is_post_gloas` check.